test/test_mps.py

```diff
@@ -8196,7 +8204,7 @@ def helper(dtype):
 
         [helper(dtype) for dtype in [torch.float32, torch.float16, torch.int32, torch.int16, torch.uint8, torch.int8]]
 
-    @unittest.skipIf(product_version < 14.0, "Skipped on MacOS < 14.0")
+    @xfailIf(product_version < 14.0)
```

Comment:
This should work on MacOS 13, shouldn't it?
```suggestion
```

Reply:
Yeah. The issue is only in the sampling with integer input on pre14 macOS. Removed the xfail from that